tests/test_samples.py

```diff
@@ -440,6 +440,10 @@ def test_plot_1d_no_axes():
     assert axes.iloc[0].get_xlabel() == 'x0'
     assert axes.iloc[1].get_xlabel() == 'x1'
     assert axes.iloc[2].get_xlabel() == 'x2'
+    axes = ns.plot_1d()
+    assert axes['logL_birth'].get_xlim()[0] > -1000
```

Comment:
Is this `xlim[0] > -1000` check future proof? 
Is it needed in the first place? 
Would a check that `ns.plot_1d()` runs without error be enough?
If we add a warning, catching a warning here might actually be nice.

Reply:
I've removed the check.